packages/next-slicezone/package-lock.json

```diff
@@ -7061,6 +7061,20 @@
 				"is-typedarray": "^1.0.0"
 			}
 		},
+		"node_modules/typescript": {
```

Comment:
there shouldn't be changes here of it might publish this package.

Reply:
The real question is why is there a lock file there 🙃

Reply:
Done